src/aerugo/error.rs

```diff
@@ -28,6 +28,10 @@ pub enum RuntimeError {
     ExecutorTaskletQueueFull,
     /// Enqueued data to a full data queue.
     DataQueueFull,
+    /// Tried to perform an operation before system initialization.
+    SystemNotInitialized,
+    /// Tried to initialize system more than once
+    SystemReinitialized,
```

Comment:
`SystemAlreadyInitialized` to be in line with `StorageAlreadyInitialized`?

Reply:
fixed